x-pack/solutions/security/packages/features/src/security/product_feature_config.ts

```diff
@@ -54,6 +54,18 @@ export const securityDefaultProductFeaturesConfig: DefaultSecurityProductFeature
       },
     },
   },
+  [ProductFeatureSecurityKey.configurations]: {
+    privileges: {
+      all: {
+        ui: ['configurations'],
+        api: [],
+      },
+      read: {
+        ui: ['configurations_read'],
+        api: [],
+      },
```

Comment:
Does this page support read-only mode?

Reply:
To be honest, no idea, is the read-only mode optional?

Reply:
Yes, this config is essentially extending the base privileges of the feature, we can add actions to the `all` or the `read` privilege, or both. 
If we are never checking the `configurations_read` capability (the page does not have a read-only mode), there's no point in extending the `read` privilege, we can just remove the key here.